. gitattributes

```diff
@@ -0,0 +1,17 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Declare files that will always have LF line endings on checkout.
+*.sh eol=lf
+
+# Explicitly declare text files you want to always be normalized and converted
```

Comment:
Do we need this? Looks like this is for `C`

Reply:
no, it was part of the standard templates.  I'll remove it